modules/mesaides/files/root_authorized_keys

```diff
@@ -0,0 +1,21 @@
+# User keys
+# dest=/tmp/keys; rm -f $dest && for user in MattiSG fpagnoux guillett; do echo \# $user keys >> $dest && curl https://github.com/$user.keys >> $dest && echo '' >> $dest; done
```

Comment:
I added this shell snippet to make sure that updates are relatively easy.
Given the rest of the file content, a template or some other logic would have been required and I preferred to rely on a _basic_ file.